src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,86 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+/*This class holds the TYDRecommended algorithm which will give the
+    user a recommended store to get their desired beverage from.
+*/
+public class Recommended {
```

Comment:
rename to RecommendationUtils or RecommendationEngine

Reply:
changed name to RecommendationEngine